pkg/policy/api/rule.go

```diff
@@ -62,6 +62,29 @@ type Rule struct {
 	Description string `json:"description,omitempty"`
 }
 
+// Entity specifies a special entity for rules
+type Entity string
+
+const (
+	// World is an entity that represents traffic external to endpoint environment
+	World Entity = "world"
+	// Host is an entity that represents traffic within endpoint host
+	Host Entity = "host"
+)
+
+var EntitySelectorMapping = map[Entity]EndpointSelector{
+	World: NewESFromLabels(&labels.Label{
+		Key:    "world",
```

Comment:
I think we should define these in `pkg/labels/` and then just reference them from here.

Reply:
done